### PR TITLE
PROJECT: Small Library app

### PR DESCRIPTION
**Pull request by macnick**
In this project we: 
1. Created basic layout using simple HTML and CSS files
2. All the logic is in the JS file
3. JS is organized in a MVC way. **Model** is the Book function constructor. **booklist** object holds all the controller functions and **view** holds the functions that draw the screen
4. There is a button to Add a new Book
5. There is a button to Delete a book
6. There is a button to Toggle the Status of a book between Read/Unread
7. Implemented localStorage to store the books even after the browser closes
8. Implemented messages when the user successfully adds, deletes a book and when validation fails

Please review our code.

### CODE REVIEW

File: assets/index.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  localStorage.setItem('books', JSON.stringify(library));
+};
+// If storage is empty we store the demo books
+if (!localStorage.length) {
+  populateStorage();
+}
+
+library = JSON.parse(localStorage.getItem('books'));
+
+const view = {
```
Comment (bellom) [inline assets/index.js:11]: Let keep the application logic separated from DOM manipulation tasks.
You move the DOM tasks in another file.
image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome Design!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just one thing to work on............. :slightly_smiling_face: 
 
**_kindly check out my comment below;_**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by macnick** [comment]
Thank you for the review. 

I moved the offending code to the proper place. 

Please review our code.

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# changes required 👀 🔍 

Hey @macnick! Hey @addod19! 👋 

The project looks fine in the live demo!
But based on changes made, there seems to be an issue when running locally.
See attached screenshot and address these issues! 🙏 
Screen Shot 2020-02-27 at 8 43 44 PM

Which causes the project to load like the following:
Screen Shot 2020-02-27 at 8 48 10 PM


***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by addod19** [comment]
It is probably a localStorage bug, a quick fix is to clear the localStorage with the command ```localStorage.clear()``` and it will work fine. Kindly review our project 🙂 .Thank you

**Review by mcrd25 — APPROVED** [review]
# changes Approved! 💪 🤓 🎊 

Hi Team!

No need for further review! All is well now!

**Comment by addod19** [comment]
> # changes Approved! 💪 🤓 🎊
> Hi Team!
> 
> No need for further review! All is well now!

Thank you for your review 👐

**Pull request merged**